26-register_registration/resource_types/artistic_result/creator_selects_resource_type_artistic_result_and_subtype_architecture.feature

```diff
@@ -0,0 +1,89 @@
+Feature: Creator selects Resource type Artistic Result and subtype Architecture
+
+  @3413
+  Scenario: Creator navigates to the Resource Type tab and selects Resource subtype "Architecture"
+    Given Creator navigates to the Resource Type tab and selects Resource type "Artistic Result"
+    When they select Resource Subtype "Architecture"
+    Then they see fields:
+      | More information |
+    And they see field for Type Work with options:
+      | Construction           |
+      | Plan proposal          |
+      | Landscape architecture |
+      | Interior               |
+      | Other                  |
+    And they see a list of Exhibitions with fields:
+      | Name  |
+      | Date  |
+      | Order |
+    And they see that each Exhibition has a Delete Button
+    And they see that each Exhibition has an Edit Button
+    And see they Buttons for:
+      | Add Competition            |
+      | Add Publication or Mention |
+      | Add Prize or Award         |
+      | Add Exhibition             |
+
+  @3414
+  Scenario: Creator adds an Competition
+    Given Creator navigates to the Resource Type tab and selects Resource subtype "Architecture"
+    When they click the Add Competition Button
```

Comment:
The focus here should be on the requirements not the implementation, it is apparent since we have a given-when-then-when-then that we struggle to define the UI via definition here. I suggest simply not doing it…the glue code should do the implementation stuff.

Given Creator navigates to the Resource Type tab and selects Resource subtype "Architecture"
When they add a competition with details for:
   | Name |
   | Description |
   | Date |
Then the Competition is listed under Exhibitions

I suppose listing a competition under exhibitions is a bit weird too, but I am not a domain expert

Reply:
Enig. Tar en runde rundt dette med @Eiriknil før jeg endrer